Makefile

```diff
@@ -0,0 +1,10 @@
+all: grpc
+
+grpc:
+	python -m grpc_tools.protoc --proto_path=quarkchain/proto --python_out=quarkchain/generated --grpc_python_out=quarkchain/generated grpc.proto
+	$(MAKE) -C qkchash
```

Comment:
should probably give qkchash a separate target?

Reply:
done